tfhe-zk-pok/src/proofs/mod.rs

```diff
@@ -359,4 +373,48 @@ mod test {
             PkeTestCiphertext { c1, c2 }
         }
     }
+
+    /// Return a point with coordinates (x, y) that is randomly chosen and not on the curve
+    pub(super) fn point_not_on_curve<Config: short_weierstrass::SWCurveConfig>(
```

Comment:
Could be in a `#[cfg(test)]` I guess
Applies to other functions

Reply:
it's already the case (it's inside `mod test`)